src/lwc/utilInput/utilInput.js

```diff
@@ -100,12 +103,25 @@ export default class utilInput extends LightningElement {
         return false;
     }
 
+    get dataType() {
+        return this.type ? this.type.toLowerCase() : undefined;
+    }
+
     get formatter() {
-        let type = checkNestedProperty(this.fieldDescribe, 'dataType') ?
-            this.fieldDescribe.dataType :
-            this.type;
-        if (type === 'Currency' || type === 'Percent' || type === 'Decimal') {
-            return type;
+        if (this.dataType === CURRENCY || this.dataType === PERCENT || this.dataType === DECIMAL) {
+            return this.dataType;
+        }
+        return undefined;
+    }
+
+    get granularity() {
```

Comment:
This looks virtually identical to the one in geFormField - any way this could be a single function that we share  between the two?